### ISSUE TITLE
Logout does not support adding BASE_URL variable value to 'redirect_uri'

### ISSUE THREAD

**Issue by brophyja** [alerta/alerta-webui#401]
**Issue Summary**
When the Alerta WebUI is being served from a base path (e.g. /alerta), logout from the WebUI does not support adding the BASE_URL environment variable value to the 'redirect_uri' that is returned to the client web browser.

**Environment**
- OS: Linux
- API version: 8.1.0
- Deployment: OpenShift/Docker
- WSGI environment:  Gunicorn

- Database: Postgres 10

- Server config:
  Auth enabled? Yes
  Auth provider? Keycloak
  Customer views? No

- web UI version: 8.0.1
- CLI version: n/a

- Additional Deployment Details:
  The API Server is run from a Python/Gunicorn container and is served on the path:  /alerta-api
  The Alerta WebUI is run from an Nginx container and is served on the path:  /alerta
  The Alerta WebUI was built from source with the variable 'BASE_PATH=/alerta' set.
  The Alerta WebUI is being run with the variable 'BASE_URL=/alerta' set.

**To Reproduce**
1. While logged into the Alerta WebUI, select the user profile icon in the upper right hand corner
2. Click on 'LOG OUT'

The session logout is correctly passed to API Server & Keycloak Server, resulting in the user session being logged out.
The 'redirect_uri' is sent back to the user web browser (Firefox) from the Keycloak Server with the incorrect 'redirect_uri' that is missing the '/alerta' path.

This is the Firefox debug output showing the incorrect 'redirect_uri' lacking the '/alerta' BASE_URL :
![alerta-logout-bad-redirect_uri](https://user-images.githubusercontent.com/44980929/99856329-9bb10080-2b3d-11eb-8ba7-9940e4a7de34.png)

**Expected behavior**
If the BASE_URL variable is set, the logout 'redirect_uri' that is returned to the client web browser should include the BASE_URL path.

**Additional context**
With Alerta WebUI [pull request 380](https://github.com/alerta/alerta-webui/pull/380), the 'redirectUri' can be extended to include the BASE_URL path where the alerta-webui is being served from. 

However, logout from the Alerta WebUI does not support adding the BASE_URL to the redirectUrl, and only sets URL to "window.location.origin + '/logout'", as seen in the below excerpt from the 'components/auth/ProfileMe.vue' file:

```
225     logout() {
226       this.$store
227         .dispatch('auth/logout')
228         .then(response => {
229           if (response.data.logoutUrl) {
230             let redirectUrl =
231               (this.$config.provider == 'keycloak'
232                 ? 'redirect_uri='
233                 : 'post_logout_redirect_url=') +
234               window.location.origin +
235               '/logout'
236             window.location.href = response.data.logoutUrl + '?' + redirectUrl
237           } else {
238             this.$router.push({ name: 'logout' })
239           }
240         })
241       this.close()
242     },
```

As a test, I modified the 'components/auth/ProfileMe.vue' file to add the BASE_URL path if the variable is set as shown below (line 234):
```
230             let redirectUrl =
231               (this.$config.provider == 'keycloak'
232                 ? 'redirect_uri='
233                 : 'post_logout_redirect_url=') +
234               (process.env.BASE_URL ? (window.location.origin + process.env.BASE_URL) : window.location.origin) +
235               '/logout'
236             window.location.href = response.data.logoutUrl + '?' + redirectUrl
```

The modified code (ugly as it is) does result in the correct 'redirect_uri' with the '/alerta' BASE_URL path being returned to the browser, as shown in the browser debug output:
![alerta-logout-modified-redirect_uri](https://user-images.githubusercontent.com/44980929/99858380-475c4f80-2b42-11eb-80ea-358b6c464511.png)

**Comment by satterly** [comment]
Should be fixed by #403 -- can you test and confirm?

**Comment by brophyja** [comment]
I can confirm that [#403](https://github.com/alerta/alerta-webui/pull/403) does fix the issue.
Thank you, @satterly!
